pkg/signer/tink.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2025 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Copied from https://github.com/sigstore/rekor/blob/c820fcaf3afdc91f0acf6824d55c1ac7df249df1/pkg/signer/tink.go
+
+package signer
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	tinkUtils "github.com/sigstore/rekor/pkg/signer/tink"
```

Comment:
Can we pull in the Tink utilities to remove the dependency on rekor?

Reply:
it's a bit strange since it already looks like we do -- isn't that pkg/signer/tink/  in this PR?

Reply:
Yes, just forgot to update the import.

pkg/signer/tink/tink.go now exists in fulcio, rekor and rekor-tiles, maybe we should just put it in sigstore/sigstore?

Reply:
Yep, that sounds good.